docs/website/02_examples/incompressibleRubberBlock.md

```diff
@@ -115,6 +118,7 @@ vtkWriter.write("iks003_incompressibleLinearElasticity");
 ```
 
 ## Takeaways
+
 - `Ikarus::AutoDiffFE` can be used to arrive at the stiffness matrix and external load vector from the energy function.
 - Helper functions are included to switch between the Lame parameters.
 - Grids from Dune can be directly incorporated within the Ikarus framework.
```

Comment:
Maybe add easy implementation of mixed finite elements possible with composite basis

Reply:
Even it is also mentoind in cantilever beam